agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/processors/TelemetryProcessorMaskingTest.java

```diff
@@ -166,7 +166,7 @@ public long getStartEpochNanos() {
     @Override
     public Attributes getAttributes() {
       return Attributes.builder()
-          .put("http.status_code", 200L)
+          .put("http.response.status_code", 200L)
           .put("http.url", httpUrl)
           .put("http.method", "GET")
```

Comment:
I think we want these to be the new attributes, to verify that the old `http.url` based telemetry processor still works
```suggestion
          .put("url.full", httpUrl)
          .put("http.request.method", "GET")
```